deployment/cdk/opensearch-service-migration/lib/network-stack.ts

```diff
@@ -131,52 +134,84 @@ export class NetworkStack extends Stack {
             });
         }
 
-        if (props.albEnabled) {
+        const needAlb = props.captureProxyServiceEnabled ||
+            props.elasticsearchServiceEnabled ||
+            props.migrationAPIEnabled ||
+            props.captureProxyServiceEnabled ||
```

Comment:
Why is this listed twice? Is is supposed to be the capture proxy es service?

Reply:
Fixed, good catch!